### ISSUE TITLE
Tests still pass after secretless.yml config changes are done

### ISSUE THREAD

**Issue by jonahx** [cyberark/secretless-broker#711]
Depends on #710 and #708

**Comment by izgeri** [comment]
Does this also depend on #712? How is this issue supposed to work? I would guess that in making the updates in #712 we would ensure that all of the test cases are passing, no?

Is this story actually about updating the secretless config that is auto-generated in the test cases?

**Comment by izgeri** [comment]
Duplicate of #708 and #712
